distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+        with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'hello world!\\n' in line:
+                    break
+
+        with popen(['dask-submit', '127.0.0.1:8788', str(invalid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'Traceback' in line:
+                    break
+
+
+def test_submit_a_distributed_script(executor_contract_script):
```

Comment:
quick update: I was able to make it this work if RemoteClient runs in its own `IOLoop`. The above issue is probably because of sharing the same `IOLoop` with Scheduler, Executor and RemoteClient. I tried doing this by running RemoteClient in isolation in a new IOLoop and things work as expected. I have the following questions:

* is there a way in tornado to run two separate IOLoops from dask-scheduler? I have been trying to do that but i dont think i have been successful.
* it also makes me wonder if `dask-remote` that kicks off the RemoteClient should be its own cli and people explicitly run side by side a scheduler

Let me know your thoughts! Thanks!